src/components/search/detailed/DetailedSearchResults.js

```diff
@@ -0,0 +1,243 @@
+/**
```

Comment:
Another know issue - Console warning when you filter detailed search by `Apps` or `Analyses` or `Data`:
`react_devtools_backend.js:2273 Material-UI: The value provided to the Tabs component is invalid.
None of the Tabs' children match with 'ANALYSES'.
You can provide one of the following values: DATA. `